db/migrate/20220119093751_remove_fraudulent_from_fraud_match.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveFraudulentFromFraudMatch < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured { remove_column :fraud_matches, :fraudulent, :boolean }
```

Comment:
How does `safety_assured` work? My reading of the docs is that you need to add `fraudulent` to the list of `ignored_columns` and deploy that first but I'm guessing there is more to it than that?

Reply:
Good point, we forgot to do this 🤦🏼‍♀️ 

Will do it now. Thanks for catching!

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/6522